recipes-go/cpu/cpu.bb

```diff
@@ -0,0 +1,13 @@
+DESCRIPTION = "cpu command in Go, inspired by the Plan 9 cpu command"
+
+LICENSE = "BSD-3-Clause" 
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/BSD-3-Clause;md5=550794465ba0ec5312d6919e203a55f9"
```

Comment:
@PLangowski please wrap it to 80 signs. Something like below will work.

```
LIC_FILES_CHKSUM = " \
    file://${COMMON_LICENSE_DIR}/BSD-3-Clause;md5=550794465ba0ec5312d6919e203a55f9 \
"
```

Reply:
done f64259079b3fe8c4fba7c5b611fd0f26c7167182

Reply:
@PLangowski please pay attention to details and use the same form as provided

Reply:
I didn't add tab because if I do, it still exceeds 80 chars. But I guess when It's an URI it doesn't matter that much?

Reply:
Now it's the same as the one you provided 1cf69eacada404c45e4c17e133e7309f4448fad4